enable/tools/drag_tool.py

```diff
@@ -197,8 +216,24 @@ def _drag_button_up(self, event):
         return False
 
     def _drag_mouse_leave(self, event):
-        if self.end_drag_on_leave and self._drag_state == "dragging":
+        if self.end_drag_on_leave:
+            # raise deprecation warning
+            msg = ("end_drag_on_leave is now deprecated as its name was "
+                  "misleading. It triggers a drag_cancel not drag_end on "
+                  "leave. Use new on_drag_end Enum trait instead.")
+            frame = inspect.currentframe().f_back
```

Comment:
You don't appear to be using `frame` anywhere. (and you won't need the `inspect` import if you remove it)